pan3d/serve_viewer.py

```diff
@@ -11,12 +11,17 @@ def serve():
     parser.add_argument("-C", "--config_path")
     parser.add_argument("-D", "--dataset")
     parser.add_argument("-P", "--pangeo", action=BooleanOptionalAction)
+    parser.add_argument("-E", "--esgf", action=BooleanOptionalAction)
     parser.add_argument("-S", "--server", action=BooleanOptionalAction)
     parser.add_argument("-d", "--debug", action=BooleanOptionalAction)
 
     args = parser.parse_args()
 
-    builder = DatasetBuilder(dataset_path=args.dataset, pangeo=args.pangeo)
+    builder = DatasetBuilder(
+        dataset=args.dataset,
+        pangeo=args.pangeo,
+        esgf=args.esgf,
```

Comment:
are __pangeo__, __esgf__ the only 2 options or can we have more down the road?

Reply:
I'm not sure at this point whether we will have more, but I'm in favor of future-proofing and making room for any new additions. I followed your suggestion to replace these multiple boolean kwargs with one list kwarg in [36683c2](https://github.com/Kitware/pan3d/pull/67/commits/36683c226871adcb2d6da6dc9792177390ac21f8).